specifyweb/frontend/js_src/lib/components/QueryBuilder/Export.tsx

```diff
@@ -100,6 +100,41 @@ export function QueryExportButtons({
     return resultToExport.join('\n');
   }
 
+  function handleSelectedResultsKml(): string {
+    const xmlBuilder = createXMLBuilder({ version: '1.0', encoding: 'utf-8' });
```

Comment:
currious what features does this library provide that native DOM APIs don't?

also, if you don't like XML much (like me), I have a handy small function for converting json to XML:
https://github.com/specify/specify7/blob/4ad26fd93656e786765c9585214c65d47db4e97c/specifyweb/frontend/js_src/lib/components/Syncer/xmlToJson.ts#L54-L71

and if you want an even simpler to generate structure, see
https://github.com/specify/specify7/blob/4ad26fd93656e786765c9585214c65d47db4e97c/specifyweb/frontend/js_src/lib/components/Syncer/fromSimpleXmlNode.ts#L12

then SimpleXmlNode can be easily produced by your code